app/models/manageiq/providers/ansible_tower/shared/automation_manager/amazon_credential.rb

```diff
@@ -23,4 +23,10 @@ module ManageIQ::Providers::AnsibleTower::Shared::AutomationManager::AmazonCrede
   }.freeze
 
   API_ATTRIBUTES = COMMON_ATTRIBUTES.merge(EXTRA_ATTRIBUTES).freeze
+
+  API_OPTIONS = {
+    :type       => 'cloud',
+    :label      => 'Amazon',
```

Comment:
I think the `'Amazon'` here (and the other labels too) needs to be inside `N_()`:

    N_('Amazon')

so that it gets into the translation catalog and can then be localized on the UI side (or whatever would
consume this info via API).